_redirects

```diff
@@ -72,3 +72,6 @@ https://openzeppelin-docs.netlify.com/* https://docs.openzeppelin.com/:splat 301
 /contracts/3.x/api/lifecycle /contracts/3.x/api/utils 301
 /contracts/3.x/api/ownership /contracts/3.x/api/access 301
 /contracts/api/deploy-ready /contracts/api/presets 301
+
+# OpenZeppelin Contracts 3.0-rc redirect
+https://docs.openzeppelin.com/contracts/3.x-rc/* https://docs.openzeppelin.com/contracts/3.x/:splat 301!
```

Comment:
Not sure why you had used full URLs. Also I don't think the redirect should be forced (`!`).

```suggestion
/contracts/3.x-rc/* /contracts/3.x/:splat 301
```

Reply:
Heh, I just copied the template from the first redirect. I didn't know we could skip the full URL.